lib/utils/perf/src/index.js

```diff
@@ -19,6 +19,37 @@ const find = _.find;
 // Setup debug log
 const debug = require('abacus-debug')('abacus-perf');
 
+const countSample = process.env.PERF_COUNT_SAMPLE ? 
+  JSON.parse(process.env.PERF_COUNT_SAMPLE) : 
+  { granularity: 'seconds', size: 10 };
+
+const getWindow = (sample) => {
+  const seconds = 1000;
```

Comment:
Do we need to calculate these constants on every invocation of getWindow?

Reply:
We call it only once, on initialization